src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -55,11 +55,10 @@ export default function SavedObjectFactory(es, kbnIndex, Promise, Private, Notif
     let customInit = config.init || _.noop;
 
     // optional search source which this object configures
-    self.searchSource = config.searchSource && new SearchSource();
+    self.searchSource = config.searchSource ? new SearchSource() : null;
```

Comment:
I disagree. The way I see it, the value of `self.searchSource` is defined and explicitly set to a "nothing" value.